package.json

```diff
@@ -58,6 +58,7 @@
         "react-dom": "^17.0.2",
         "react-router": "^6.3.0",
         "react-router-dom": "^6.3.0",
+        "resolve-url-loader": "^5.0.0",
         "style-loader": "^3.3.1"
```

Comment:
Any webpack loaders should be moved to dev-dependencies as they are not needed in production.

Reply:
Ok!